packages/php/blueprint/src/Importers/ImportInstallTheme.php

```diff
@@ -81,17 +87,25 @@ public function process( $schema ): StepProcessorResult {
 			$this->result->add_error( "Failed to install theme '$theme->slug'." );
 		}
 
-		$theme_switch = true === $theme->activate && $this->wp_switch_theme( $theme->slug );
-
-		if ( $theme_switch ) {
-			$this->result->add_info( "Switched theme to '$theme->slug'." );
-		} else {
-			$this->result->add_error( "Failed to switch theme to '$theme->slug'." );
-		}
+		$this->activate_theme( $schema );
 
 		return $this->result;
 	}
 
+	protected function activate_theme( $schema ) {
```

Comment:
Can we also add doc comment for this function?

Reply:
Done in [Add comment and fix lint issue](https://github.com/woocommerce/woocommerce/pull/54586/commits/40faaabd1cc70e7475272633ea25dbd53fbb4e49)